src/igl/vulkan/Common.cpp

```diff
@@ -156,69 +156,69 @@ VkFormat textureFormatToVkFormat(igl::TextureFormat format) {
   case TextureFormat::RGBA_ASTC_6x5:
-    return VK_FORMAT_ASTC_6x5_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_6x5_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_6x5:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_6x5_SRGB_BLOCK;
   case TextureFormat::RGBA_ASTC_6x6:
-    return VK_FORMAT_ASTC_6x6_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_6x6_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_6x6:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_6x6_SRGB_BLOCK;
   case TextureFormat::RGBA_ASTC_8x5:
-    return VK_FORMAT_ASTC_8x5_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_8x5_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_8x5:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_8x5_SRGB_BLOCK;
   case TextureFormat::RGBA_ASTC_8x6:
-    return VK_FORMAT_ASTC_8x6_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_8x6_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_8x6:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_8x6_SRGB_BLOCK;
   case TextureFormat::RGBA_ASTC_8x8:
-    return VK_FORMAT_ASTC_8x8_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_8x8_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_8x8:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_8x8_SRGB_BLOCK;
   case TextureFormat::RGBA_ASTC_10x5:
-    return VK_FORMAT_ASTC_10x5_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_10x5_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_10x5:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_10x5_SRGB_BLOCK;
   case TextureFormat::RGBA_ASTC_10x6:
-    return VK_FORMAT_ASTC_10x6_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_10x6_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_10x6:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_10x6_SRGB_BLOCK;
   case TextureFormat::RGBA_ASTC_10x8:
-    return VK_FORMAT_ASTC_10x8_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_10x8_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_10x8:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_10x8_SRGB_BLOCK;
   case TextureFormat::RGBA_ASTC_10x10:
-    return VK_FORMAT_ASTC_10x10_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_10x10_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_10x10:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_10x10_SRGB_BLOCK;
   case TextureFormat::RGBA_ASTC_12x10:
-    return VK_FORMAT_ASTC_12x10_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_12x10_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_12x10:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_12x10_SRGB_BLOCK;
   case TextureFormat::RGBA_ASTC_12x12:
-    return VK_FORMAT_ASTC_12x12_SRGB_BLOCK;
+    return VK_FORMAT_ASTC_12x12_UNORM_BLOCK;
   case TextureFormat::SRGB8_A8_ASTC_12x12:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_ASTC_12x12_SRGB_BLOCK;
   case TextureFormat::RGBA_PVRTC_2BPPV1:
-    return VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG;
+    return VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG;
   case TextureFormat::RGB_PVRTC_2BPPV1:
-    return VK_FORMAT_UNDEFINED;
+    return VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG;
```

Comment:
Shouldn't it be `VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG`?

Reply:
Yes, you are right.

https://github.com/KhronosGroup/Vulkan-Docs/issues/512

IMG_texture_compression_pvrtc

GL_COMPRESSED_RGB_PVRTC_4BPPV1_IMG -> VK_FORMAT_PVRTC1_4BPP_UNORM_BLOCK_IMG
GL_COMPRESSED_RGB_PVRTC_2BPPV1_IMG -> VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG
GL_COMPRESSED_RGBA_PVRTC_4BPPV1_IMG -> VK_FORMAT_PVRTC1_4BPP_UNORM_BLOCK_IMG
GL_COMPRESSED_RGBA_PVRTC_2BPPV1_IMG -> VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG
IMG_texture_compression_pvrtc2

GL_COMPRESSED_RGBA_PVRTC_2BPPV2_IMG -> VK_FORMAT_PVRTC2_2BPP_UNORM_BLOCK_IMG
GL_COMPRESSED_RGBA_PVRTC_4BPPV2_IMG -> VK_FORMAT_PVRTC2_4BPP_UNORM_BLOCK_IMG
GL_COMPRESSED_RGBA_PVRTC_2BPPV2_IMG -> VK_FORMAT_PVRTC2_2BPP_UNORM_BLOCK_IMG
GL_COMPRESSED_RGBA_PVRTC_4BPPV2_IMG -> VK_FORMAT_PVRTC2_4BPP_UNORM_BLOCK_IMG
EXT_pvrtc_sRGB

GL_COMPRESSED_SRGB_PVRTC_2BPPV1_EXT -> VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG

GL_COMPRESSED_SRGB_PVRTC_4BPPV1_EXT -> VK_FORMAT_PVRTC1_4BPP_SRGB_BLOCK_IMG

GL_COMPRESSED_SRGB_ALPHA_PVRTC_2BPPV1_EXT -> VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG

GL_COMPRESSED_SRGB_ALPHA_PVRTC_4BPPV1_EXT -> VK_FORMAT_PVRTC1_4BPP_SRGB_BLOCK_IMG

GL_COMPRESSED_SRGB_ALPHA_PVRTC_2BPPV2_IMG -> VK_FORMAT_PVRTC2_2BPP_SRGB_BLOCK_IMG

GL_COMPRESSED_SRGB_ALPHA_PVRTC_4BPPV2_IMG -> VK_FORMAT_PVRTC2_4BPP_SRGB_BLOCK_IMG